### PR TITLE
Standalone agent fixes

### PR DESCRIPTION
**Pull request by Haishi2016**
1. Updated MQTT provider to use thread-safe lock map to maintain locks per request/response pair instead of request type.
2. Script provider to use different temporary output file names for Get() and Apply() requests.
3. Maestro CLI to support a new agent subcommand to launch a Symphony agent. Needs to extend on this more to set up a system daemon.
4. Updated opera portal to use new logo and minor display improvements for targets.

### CODE REVIEW

File: api/pkg/apis/v1alpha1/providers/target/script/script.go [hunk 1]
```diff
@@ -400,13 +400,7 @@ func (i *ScriptProvider) Apply(ctx context.Context, deployment model.DeploymentS
...
 		metrics.DeleteOperationType,
 	)
-	providerOperationMetrics.ProviderOperationLatency(
-		applyTime,
-		script,
-		functionName,
-		metrics.ApplyOperation,
-		metrics.UpdateOperationType,
-	)
+
```
Comment (msftcoderdjw) [inline api/pkg/apis/v1alpha1/providers/target/script/script.go:403]: Why we removed the metrics here?

File: api/pkg/apis/v1alpha1/providers/target/mqtt/mqtt.go [hunk 2]
```diff
@@ -59,15 +59,11 @@ type ProxyResponse struct {
...
-	Config          MQTTTargetProviderConfig
-	Context         *contexts.ManagerContext
-	MQTTClient      gmqtt.Client
-	GetChan         chan ProxyResponse
-	RemoveChan      chan ProxyResponse
-	NeedsUpdateChan chan ProxyResponse
-	NeedsRemoveChan chan ProxyResponse
-	ApplyChan       chan ProxyResponse
-	Initialized     bool
+	Config        MQTTTargetProviderConfig
```
Comment (msftcoderdjw) [inline api/pkg/apis/v1alpha1/providers/target/mqtt/mqtt.go:62]: @LeiQL @yanjiaxin534 please help to review this PR for providers change.

File: coa/pkg/apis/v1alpha2/bindings/mqtt/mqtt.go [hunk 3]
```diff
@@ -72,21 +72,23 @@ func (m *MQTTBinding) Launch(config MQTTBindingConfig, endpoints []v1alpha2.Endp
 			response = routeTable[request.Route].Handler(request)
 		}
 
-		// needs to carry call-context from request into response
+		// needs to carry request-id from request into response
```
Comment (RemindD) [inline coa/pkg/apis/v1alpha2/bindings/mqtt/mqtt.go:75]: Not related to this change - If there are multiple replicas of symphony, will subscribers in mqtt binding of different replicas process the same request multiple times?

File: api/pkg/apis/v1alpha1/providers/target/script/script.go [hunk 4]
```diff
@@ -202,7 +202,7 @@ func (i *ScriptProvider) Get(ctx context.Context, deployment model.DeploymentSpe
 	id := uuid.New().String()
 	input := id + ".json"
 	input_ref := id + "-ref.json"
-	output := id + "-output.json"
+	output := id + "-get-output.json"
```
Comment (LeiQL) [inline api/pkg/apis/v1alpha1/providers/target/script/script.go:205]: I see script ut failed due to the changed output. Please help fix it.

File: coa/pkg/apis/v1alpha2/bindings/mqtt/mqtt.go (hunk 3 above)
Comment (Haishi2016) [reply coa/pkg/apis/v1alpha2/bindings/mqtt/mqtt.go:75]: Indeed. We need to work out the scale-out situation.

File: api/pkg/apis/v1alpha1/providers/target/script/script.go (hunk 4 above)
Comment (Haishi2016) [reply api/pkg/apis/v1alpha1/providers/target/script/script.go:205]: Good catch. Fixed.

File: api/pkg/apis/v1alpha1/providers/target/script/script.go (hunk 1 above)
Comment (Haishi2016) [reply api/pkg/apis/v1alpha1/providers/target/script/script.go]: By accident. Restored.

**Comment by msftcoderdjw** [comment]
@Haishi2016 , there're some conflicts. Would you please rebase once?

**Comment by Haishi2016** [comment]
> @Haishi2016 , there're some conflicts. Would you please rebase once?

This is because I ran an experimental release off this branch and version got bumped. I've changed them back.

**Review by msftcoderdjw — APPROVED** [review]

**Pull request merged**